server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -92,8 +93,32 @@ private void addServices(ServerBuilder sb) {
 
   public static void main(String[] args) {
     int port = 8080;
-    if (args.length > 0) {
-      port = Integer.parseInt(args[0]);
+    // Create Options object
```

Comment:
Avoid comments on each line, the code speaks for itself :bowtie:. This applies for the comments below as well.

Reply:
Okay - I was following the same approach used [here]( https://github.com/unitycatalog/unitycatalog/blob/043dd3217fe5f6c5db36e9b4b240f3b8243ecb9c/examples/cli/src/main/java/io/unitycatalog/cli/UnityCatalogCli.java#L26). I will remove them.